slack_sdk/oauth/installation_store/sqlalchemy/__init__.py

```diff
@@ -48,8 +51,11 @@ def build_installations_table(cls, metadata: MetaData, table_name: str) -> Table
             Column("user_token", String),
             Column("user_scopes", String),
             Column("incoming_webhook_url", String),
+            Column("incoming_webhook_channel", String),
             Column("incoming_webhook_channel_id", String),
             Column("incoming_webhook_configuration_url", String),
+            Column("is_enterprise_install", String),
```

Comment:
Didn't Boolean work here?
```suggestion
            Column("is_enterprise_install", Boolean, default=False, nullable=False),
```

Reply:
Looks like it does, just had to add a import up top

```
from sqlalchemy.sql.sqltypes import Boolean
```